lib/hexpm_web/controllers/versions_controller.ex

```diff
@@ -0,0 +1,28 @@
+defmodule HexpmWeb.VersionsController do
```

Comment:
nitpick: by convention we keep Web module names singular, could you change to VersionController and VersionView? I Missed this when reviewing the 1st PR.

Reply:
Maybe this should actually be `HexpmWeb.ReleaseController` to match [`Hexpm.API.ReleaseController`](https://github.com/hexpm/hexpm/blob/49bd6d2f8a6a8c9dc3eefd65ea4ef169c337fb1e/lib/hexpm_web/controllers/api/release_controller.ex)? I think it's fine as is but curious what you think. cc @ericmj

Reply:
I have a really hard time knowing where to split controllers. I'm not sure it matches `Hexpm.API.ReleaseController` because it doesn't look like it has equivalent actions to this controller, but I would trust Wojtek's judgement over mine in this area.

Reply:
you have a point that the actions don't overlap; let's stick to version but use singular name.